pkgs/by-name/op/openfortivpn-webview/package.nix

```diff
@@ -0,0 +1,34 @@
+{ lib
+, stdenv
+, fetchurl
+}:
+
+let
+  pname = "openfortivpn-webview";
+  version = "1.2.0";
+in
+
+stdenv.mkDerivation {
+  inherit pname version;
+
+  src = fetchurl {
+    url = "https://github.com/gm-vm/${pname}/releases/download/v${version}-electron/${pname}-${version}.tar.xz";
```

Comment:
Since this is a release on Github, you should use the `fetchFromGithub` fetcher instead: https://nix.dev/tutorials/packaging-existing-software.html#fetching-source-from-github